src/Field.js

```diff
@@ -30,6 +31,11 @@ const createField = ({ deepEqual, getIn, setIn }) => {
         this.ConnectedField =
           createConnectedField(this.context._reduxForm, { deepEqual, getIn }, nextProps.name)
       }
+
+      const syncError = this.getSyncError()
+      if (syncError !== this.state.syncError) {
+        this.setState({ syncError })
+      }
```

Comment:
I think it is a problem that can not track error properly. 
If field`s validation depend on itself, it is easy to track the error.
If field's validation depend on other field, there is no way to track the error unless make component  rerender every time.